deebotozmo/commands.py

```diff
@@ -29,6 +29,23 @@ def __init__(self, speed: str):
         })
 
 
+class SetScheduleOnce(Command):
+
+    def __init__(self, hour: str, minute: str ):
+        super().__init__("setSched", {
+            "act": "add",
+            "content": {"jsonStr":"{\"type\":\"auto\"}","name":"clean"},
+            "enable": 1,
+            "hour": int(hour),
+            "index": 2,
+            "mid":"1037360161",
```

Comment:
```suggestion
            "mid": mid,
```
Otherwise this command will only work for your setup

Reply:
How i get the mid?

Reply:
cachedmapinfo will return all mid

Reply:
Ok, it is located in map.py but how can i call mid from commands.py?

Reply:
mid will be a parameter. The caller must pass it in, add `mid` to the init method

Reply:
I`m sure i tried it. Will test again.